boards/opentitan/README.md

```diff
@@ -20,7 +20,7 @@ Programming
 -----------
 
 Tock on OpenTitan requires
-lowRISC/opentitan@199d45626f8a7ae2aef5d9ff73793bf9a4233711 or newer. In
+lowRISC/opentitan@217a0168ba118503c166a9587819e3811eeb0c0c or newer. In
```

Comment:
Lets get things working closer to HEAD than this commit.  Once we get everything (or most things) working satisfactorily, please consult with the opentitan project about which commit to use.

Reply:
When I started this was one of the latest commits. I think it's best to just get this one working then we can go newer again after that

Reply:
In its current state , this PR will not work at opentitan HEAD (because of the [test_rom changes I mentioned](https://github.com/alistair23/tock/pull/1#discussion_r931743828)).

The PR you chose is before we made the test rom compliant with the [manifest definition](https://docs.opentitan.org/sw/device/silicon_creator/rom_ext/docs/manifest/) and is nearly 60 days out of date.